src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -146,6 +142,11 @@ export const MyProfile = () => {
                           {t('common.save')}
                         </LoadingButton>
                       </Box>
+                      <Divider sx={{ my: '1rem' }} />
+                      <Box sx={{ display: 'flex', flexDirection: 'column', mt: '0.5rem' }}>
+                        <Typography fontWeight={600}>{t('common.orcid')}</Typography>
+                        <UserOrcid user={user} />
+                      </Box>
```

Comment:
Denne kan fjernes nå? Hvis ikke er det bare å merge
```suggestion
                        <Typography fontWeight={600}>{t('common.orcid')}</Typography>
                        <UserOrcid user={user} />
```